libs/fluid-elements/switch/src/lib/switch.spec.ts

```diff
@@ -150,6 +150,27 @@ describe('Fluid switch', () => {
     });
   });
 
+  describe('value', () => {
+    it('should set the default value to on', async () => {
```

Comment:
please add a test scenario where no value is present

Reply:
This is the test where no value is present. The default based on the MdnDocumentation is on.
@rowa-audil maybe reword the tests it to should have the correct default set.